charts/nfs-server-provisioner/values.yaml

```diff
@@ -8,7 +8,7 @@ replicaCount: 1
 
 image:
   repository: registry.k8s.io/sig-storage/nfs-provisioner
-  tag: v4.0.8
+  tag: v6.1
```

Comment:
A new tag has landed, 6.2, that might fix a critical functionality issue when using Linux kernel >= 6.6 https://github.com/nfs-ganesha/nfs-ganesha/issues/1188

Reply:
OK, I'll check this towards newer Ganesha then.

Reply:
Thanks.